src/components/TreeMenu/index.js

```diff
@@ -21,26 +21,26 @@ const TreeMenu = ({
 			>
 				{({ search, items }) => (
 					<>
-						{ props.hasSearch && (
-							<InputGroup>
-								<InputGroupText className="p-0 border-0">
-									{searchOptions?.dropdown && (
-										<ButtonDropdown
-											size="sm"
-											isOpen={isDropdownMenuOpen}
-											toggle={() => setDropdownMenu(prev => !prev)}
-										>
-											<DropdownToggle caret>{searchOptions.dropdown.title}</DropdownToggle>
-											{searchOptions.dropdown.children}
-										</ButtonDropdown>
-									)}
-								</InputGroupText>
-								<Input
-									onChange={e => search(e.target.value)}
-									placeholder={searchOptions?.placeholder}
-								/>
-							</InputGroup>
-						)}
+						<InputGroup>
```

Comment:
@Samsunggora This looks like you were doing some "magic" with rebasing/merging master branch into yours because of conflicting branches. Please be patient and avoid that, since those are not changes which should be a part of this PR (and I know that those are already in master)